tailwind.config.ts

```diff
@@ -58,6 +58,11 @@ export default {
         },
         white: colors.white,
         transparent: colors.transparent,
+        primaryGradient: {
+          first: lightDark(0.7985, 0.132, 159.5), // #67D79F
```

Comment:
It's not an issue, but no screen is able to make a difference between 0.7985 & 0.799. There is no reason to be that specific ^^

Reply:
Makes sense! The reason to add it here was to make it easier for forks to find these colors and customize them without having to change the buttonStyles, and I thought might as well use lightDark too